docs-chef-io/content/inspec/resources/aws_vpn_connection.md

```diff
@@ -66,7 +61,15 @@ There are also additional properties available. For a comprehensive list, see [t
 ### Test that a VPN Connection is available
 
 ```ruby
-describe aws_vpn_connection('vpn-01234567') do
+describe aws_vpn_connection('vpn-1234567890') do
+    its('vpn_connection_id') { should eq 'vpn-1234567890' }
+end
+```
+
+### Test that a VPN Connection status is `available`
+
+```ruby
+describe aws_vpn_connection('vpn-1234567890') do
     its('status') { should eq 'available' }
 end
 ```
```

Comment:
Both the test are same rite? Please check once.

Reply:
One test is checking for id and another test is checking for status.

Reply:
ok